x-pack/plugins/apm/public/components/app/correlations/failed_transactions_correlations_help_popover.tsx

```diff
@@ -36,7 +36,7 @@ export function FailedTransactionsCorrelationsHelpPopover() {
       <p>
         <FormattedMessage
           id="xpack.apm.correlations.failedTransactions.helpPopover.basicExplanation"
-          defaultMessage="Correlations help you discover which attributes are contributing to failed transactions. Transactions are considered a failure when it returns a status code >= 5xx."
+          defaultMessage="Correlations help you discover which attributes are most influential in distinguishing between transaction failures and successes. Transactions are considered a failure when their `event.outcome` value is `failure`."
```

Comment:
By default transactions with status code 5xx are marked as failed. So it might make sense to still mention this somehow (or maybe this belongs in the docs?)

Reply:
Thanks!  I've added that example to the docs in https://github.com/elastic/kibana/pull/111219